main/relational_to_grid.es6.js

```diff
@@ -0,0 +1,115 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+const path = require('path');
+const process = require('process');
+const url = require('url');
+
+require('foam2');
+
+require('../lib/grid_dao.es6.js');
+require('../lib/http_json_dao.es6.js');
+require('../lib/json_dao_container.es6.js');
+require('../lib/local_json_dao.es6.js');
+require('../lib/web_apis/release.es6.js');
+require('../lib/web_apis/release_interface_relationship.es6.js');
+require('../lib/web_apis/web_interface.es6.js');
+const pkg = org.chromium.apis.web;
+
+const logger = foam.log.ConsoleLogger.create();
+
+const USAGE = `USAGE:
+
+    node /path/to/import.es6.js BaseConfluenceDataURL
+
+        BaseConfluenceDataURL = absolute https: or file: URL to directory
+                                where Confluence data are stored with NO
+                                trailing slash.`;
+if (process.argv.length !== 3) {
+  console.error(USAGE);
+  process.exit(1);
+}
+
+const dataUrl = url.parse(process.argv[2]);
+if (dataUrl.protocol !== 'file:' && dataUrl.protocol !== 'https:') {
```

Comment:
Because a spec that says 'append character : to every string returned by this method' is a good spec, right?

Or, just another wat from JS-land.

Reply:
I'm not sure I understand the question. URLs could have other schemes like `http:` or `chrome:`. We don't want to attempt to use them in this script.

Reply:
Sorry, just commenting on silly things that JS does. The spec for this field says that all implementations must include a (useless) colon to the end of the scheme string.